src/Workspaces/Core/Portable/Workspace/Solution/Document.cs

```diff
@@ -257,6 +262,37 @@ public bool TryGetSemanticModel([NotNullWhen(returnValue: true)] out SemanticMod
             return _model != null && _model.TryGetTarget(out semanticModel);
         }
 
+        /// <summary>
+        /// Gets the current nullable disabled semantic model for this document if the model is already computed and still cached.
+        /// In almost all cases, you should call <see cref="GetSemanticModelAsync(CancellationToken)"/>, which will compute the semantic model
+        /// if necessary.
+        /// </summary>
+        [Experimental(RoslynExperiments.NullableDisabledSemanticModel, UrlFormat = RoslynExperiments.NullableDisabledSemanticModel_Url)]
+        public bool TryGetNullableDisabledSemanticModel([NotNullWhen(returnValue: true)] out SemanticModel? semanticModel)
+        {
+            semanticModel = null;
+            return _nullableDisabledModel != null && _nullableDisabledModel.TryGetTarget(out semanticModel);
+        }
```

Comment:
who calls this method?

Reply:
'GetRequiredNullableDisabledSemanticModelAsync' in 'DocumentExtensions.cs' calls it as well as 'GetSemanticModelHelperAsync' in 'Document.cs'